packages/fast-usdc/src/fast-usdc.contract.js

```diff
@@ -160,6 +173,19 @@ export const contract = async (zcf, privateArgs, zone, tools) => {
         );
       });
     },
+    async publishAddresses() {
+      const [poolAccountAddress, settlementAccountAddress] =
+        await vowTools.when(
+          vowTools.all([
+            E(poolAccount).getAddress(),
+            E(settlementAccount).getAddress(),
+          ]),
+        );
+      await publishAddresses(storageNode, {
```

Comment:
`publishAddresses` calling a _different_ `publishAddresses` is quite confusing. I first thought this should be inlined but I imagine you wanted some documentation and type annotation of the helper.

Usually we would get that with a `RecorderKit`. Then I thought it's not necessary in this contract because the "on-chain can read same values as off-chain" requirement isn't pertinent to this contract that only works off-chain.

But thinking about it more, it's imaginable that another contract would want a reference to these accounts. E.g. to automate pool participation.

So I think an `addressRecorderKit.recorder.write` should be used to write these values. A `makeRecorderKit` is already available in scope.

Reply:
Good feedback, contracts wanting to consume this value crossed my mind over the weekend but I forgot about it when revisiting the changes today.

As an alternative to a recorderKit, what do you think about a `getAddresses` public facet method? This might be more fitting since we expect the values to be static, or at least append only?

Reply:
Good point. Since it never changes a RecorderKit is overkill. To emphasize the static nature consider `getStaticInfo()` or something like that where we could include all static info about the contract